### PR TITLE
Remove duplicate of NativePHP link

### PR DESCRIPTION
**Pull request by fiedsch**

### CODE REVIEW

File: docs/blog_post.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ I also don't plan to add support for other databases, because SQLite is everywhe
 register your own SQL functions, which Loupe benefits from.
 
 Thanks to SQLite, Loupe could also become interesting for embedded systems such as apps built with [NativePHP]
```
Comment (Toflar) [inline docs/blog_post.md:92]:
```suggestion
Thanks to SQLite, Loupe could also become interesting for embedded systems such as apps built with [NativePHP][NativePHP]
```

Comment (Toflar) [reply docs/blog_post.md:92]: You need it to be duplicated - the issue is just the added new line.

File: docs/blog_post.md [hunk 2]
```diff
@@ -146,4 +146,4 @@ Yanick
 [State_Set_Index]: https://github.com/Toflar/state-set-index
 [GitHub_Sponsor]: https://github.com/sponsors/Toflar
 [Mastodon]: https://phpc.social/@toflar
-[Twitter]: https://twitter.com/toflar
\ No newline at end of file
+[Twitter]: https://twitter.com/toflar
```
Comment (Toflar) [inline docs/blog_post.md:149]: This is not needed.

File: docs/blog_post.md (hunk 1 above)
Comment (fiedsch) [reply docs/blog_post.md:92]: Sorry, the I must have messed it somehow. I read the aarticlw yesterday and was confused by (what looked to me as a) duplicate link. I will check again and then probably close the issue. Sorry for the confusion.

**Pull request closed without merging**